esx_service/cli/vmdkops_admin.py

```diff
@@ -304,10 +307,13 @@ def commands():
                                     'help': "Datastore which access is controlled",
                                     'required': True
                                 },
-                                '--rights': {
-                                    'help': 'Datastore access Permissions granted',
-                                    'choices': ['create', 'delete', 'mount', 'all'],
-                                    'metavar': 'create,delete,mount'
+                                '--switch-default-datastore': {
+                                    'help': "Switch default datastore for this tenant",
+                                    'action': 'store_true'
+                                },                            
+                                '--allow-create': {
+                                    'help': 'Allow create and delete on datastore if set to True',
+                                    'action': 'store_true'
```

Comment:
Let us keep it as is for now.  We can make the change later if user complains about it.